kotlintest-assertions/kotlintest-assertions-arrow/build.gradle

```diff
@@ -23,4 +23,11 @@ dependencies {
     testCompile project(":kotlintest-runner:kotlintest-runner-junit5")
 }
 
+compileKotlin {
+    kotlinOptions {
+        freeCompilerArgs += '-Xuse-experimental=kotlin.Experimental'
```

Comment:
Is this actually necessary?

I remember adding a experimental contract somewhere else in our code, and I don't think I needed to add this.

Reply:
Yeah you need it where the contract is defined, but not where it is used.